src/services/userService.js

```diff
@@ -0,0 +1,45 @@
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getAllUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
+  return users.find((person) => person.id === +id) || null;
+};
+
+const createUser = (name) => {
+  const user = {
+    id: Math.floor(Math.random() * 1000),
```

Comment:
The way you are generating user IDs can potentially lead to duplicate IDs. This is because the `Math.random()` function can return the same value more than once. It's recommended to use a more reliable way of generating unique IDs. One popular method is to use libraries like `uuid`.